Caboodle/Compass/Compass.shared.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Compass
+    {
+        public static bool IsMonitoring =>
+            MonitorCTS != null && !MonitorCTS.IsCancellationRequested;
```

Comment:
_super_ picky and doesn't truly matter, but is:  `MonitorCTS?.IsCancellationRequested ?? false` more concise?

Reply:
would be if it is not requested and I couldn't figure it out in my mind...

It is if Null or !IsCancellationRequested....

Reply:
I do this `MonitorCTS?.IsCancellationRequested == true` so that `false != true`, `null != true` but `true == true`

Reply:
#resolved

Reply:
I actually don't think that is right as it needs to return false if IsCancellationRequested == true :) 

How I have it is much more readable.